releasing/README.md

```diff
@@ -96,50 +78,58 @@ GITHUB_TOKEN=deadbeefdeadbeef
 echo $GITHUB_TOKEN | gh auth login --scopes repo --with-token
 ```
 
+
+Instructions on how to get a new OpenAPI sample can be found in the
```

Comment:
Shouldn't this be under `Consider fetching new OpenAPI data`? Why move it?

Reply:
Definitely. I'm not sure how I managed, but it's some sort of copy-paste error 😅